app/src/main/res/layout/drag_drop_interaction_items.xml

```diff
@@ -0,0 +1,39 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <data>
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.player.state.itemviewmodel.DragDropInteractionContentViewModel" />
+  </data>
+
+  <LinearLayout
+    android:id="@+id/drag_drop_item_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="4dp"
+    android:gravity="center_vertical">
+
+    <TextView
+      android:id="@+id/drag_drop_content_text_view"
```

Comment:
Per our discussion, are you planning to introduce the recycler view version of this in a follow up PR? If so, please add a TODO here to track that.

Reply:
recyclerview will be part of the nest PR in which we are adding support for grouping because that's the reason we need a recyclerview.